src/components/Logo/index.js

```diff
@@ -0,0 +1,81 @@
+import { RichTypography } from "@commons-ui/core";
```

Comment:
1. Have we pinged Piero to get real logo yet?
2. Logo needs to be clickable linking back to home page

Reply:
@kilemensi Pinged him on trello...waiting on a response

Reply:
@kilemensi For the search, they have the same  functionality. I'll clean that up so we have one search